docs/deployment/manifest-format.md

```diff
@@ -44,6 +44,23 @@ info: Aspire.Hosting.Publishing.ManifestPublisher[0]
 
 The file generated is the .NET Aspire manifest and is used by tools to support deploying into target cloud environments.
 
+> [!NOTE]
```

Comment:
@mitchdenny do we want the `dotnet run` command or the `dotnet msbuild /t:GenerateAspireManifest` command is the preference?

Reply:
I'm not sure if there is a preference one way or the other. However, it's ran with the appropriate args I suppose. I just added this as a note, since it felt like a convenient way to do it and I saw it in the Aspire repo.